values.yaml

```diff
@@ -253,3 +253,21 @@ serviceMonitor:
   scrapeTimeout: 10s
   # -- prometheus metrics port exposed by krakend
   targetPort: 9091
+
+# -- (object) The networkPolicies configures ingress NetworkPolicies for your application
+networkPolicies:
+  # -- (bool) Set to true to create NetworkPolicies for your application
+  enabled: false
+  # -- (list) List of NetworkPolicy objects
+  policies:
+    # -- Name of the NetworkPolicy
+    - name: "allow-namespace1"
+      # -- Name of the namespace allowed to reach (or be reached by) your application
+      namespace: "namespace1"
+      # -- Type of the network policy (ingress or egress)
+      type: "ingress"
+    - name: "allow-namespace2"
+      # -- Name of the namespace allowed to reach (or be reached by) your application
+      namespace: "namespace2"
+      # -- Type of the network policy (ingress or egress)
+      type: "egress"
```

Comment:
@ADUDevEx can we add these as a comment? This seems relevant as documentation or an example more than a default value we could provide to users. It's fine if it's empty by default. Otherwise the PR LGTM

Reply:
Hi @JAORMX , thanks for the review, I commented the namespace2 values and regenerated README.
Let me know if that's good for you :)